packages/story-editor/src/components/library/panes/shopping/shoppingPane.js

```diff
@@ -74,12 +75,15 @@ function ShoppingPane(props) {
     actions: { getProducts },
   } = useAPI();
 
+  const isShoppingEnabled =
```

Comment:
Looks like a different check in the `libraryProvider.js` ?
```
const isShoppingEnabled =
    ('none' !== shoppingProvider && isShoppingIntegrationEnabled) ||
    currentPageProducts?.length > 0;
    ```

Reply:
By design. We need this panel to render if there are products on the current page.